src/commands/users/kaokaiToday.ts

```diff
@@ -0,0 +1,131 @@
+import { CommandInteraction } from 'discord.js'
+
+import cheerio from 'cheerio'
+
+import { CommandHandlerConfig } from '../../types/CommandHandlerConfig.js'
+
+// Fetch HTML Func
+async function fetchHTML(url: string): Promise<string> {
+  try {
+    const response = await fetch(url)
+    return await response.text()
+  } catch (error) {
+    console.error('Error fetching HTML:', error)
+    throw error
+  }
```

Comment:
This fetch does not cover the case where fetched result return non-200 HTTP code.
```suggestion
return fetch(url).then(response => {
  // if response is 2xx then let it pass
  if (response.ok)
    return response.text()
  // error thrown from fetch will be full Response object which could be much better to debug in the future
  else
    throw response
})
```